README.md

```diff
@@ -16,53 +16,62 @@ The Azure Functions Core Tools provide a local development experience for creati
 
 **v2** (master branch): Self-contained cross-platform package
 
-**v3**: Self-contained cross-platform package
+**v3**: (v3.x branch): Self-contained cross-platform package **(recommended)**
 
 ## Installing
 
 ### Windows
 
-To install runtime with npm:
+To download and install with MSI:
 
-**v2**
-```bash
-npm i -g azure-functions-core-tools@2 --unsafe-perm true
-```
+**v3**
+
+- [Windows 64-bit](https://go.microsoft.com/fwlink/?linkid=2135274) - Required for VS Code debugging
```

Comment:
I think we should move `64 bit required for VS Code debugging` to a note at the top, as we also have the same note next to choco. And looking at it here, it may give the impression you need to install via MSI 64 bit for VS Code debugging (over say `npm` or `apt` or `choco`).

Reply:
Do we have anyone who would need x86? Wonder if our instructions should just be x64.

Reply:
hmm. I think in Azure we run 32 bit Host binaries, so to avoid disparity I think it makes sense to have x86 instructions.

Reply:
Yes, we should have both here.

Reply:
Updated the message to clarify that VS Code requires 64-bit version. Should be clearer now. If we say upfront that 64-bit is needed for VS Code debugging, people might miss it, or they see it and wonder how to get 64-bit from npm.